api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/QualityImprovement2020CohortQueries.java

```diff
@@ -1705,7 +1705,7 @@ public CohortDefinition getMQC13P3DEN(String indicatorFlag) {
             hivMetadata.getYesConcept().getConceptId());
 
     CohortDefinition transferredIn =
-        qualityImprovement2020Queries.getTransferredInPatients(
+        QualityImprovement2020Queries.getTransferredInPatients(
```

Comment:
Yap, this is the correct way of accessing the static objects from a class

Reply:
Great